File: output/gpt_4o/avatar/direct_translation/java/python/codeforces_99_A.py

str = list(input())
dot_index = 0
for i in range(len(str)):
    if str[i] == '.':
        dot_index = i
        break
if str[dot_index - 1] == '9':
    print("GOTO Vasilisa.")
elif int(str[dot_index + 1]) > 4:
    _str = ""
    for i in range(dot_index):
        _str += str[i]
    print(int(_str) + 1)
else:
    for i in range(dot_index):
        print(str[i], end='')
    print()

File: output/deepseek_chat/avatar/algo_based_translation/python/java/codeforces_99_A.java
import java.util.Scanner;

public class codeforces_99_A {
    public static void main(String[] args) {
        Scanner scanner = new Scanner(System.in);
        String input = scanner.nextLine();
        String[] l = input.split("\\.");
        char[] s = l[0].toCharArray();
        char[] p = l[1].toCharArray();
        int i = Character.getNumericValue(p[0]);
        
        if (s[s.length - 1] == '9') {
            System.out.println("GOTO Vasilisa.");
        } else if (s[s.length - 1] != '9' && i < 5) {
            System.out.println(String.valueOf(s));
        } else {
            String sStr = String.valueOf(s);
            int num = Integer.parseInt(sStr);
            num++;
            System.out.println(String.valueOf(num));
        }
    }
}
